x-pack/plugins/cases/server/client/cases/create.ts

```diff
@@ -27,6 +32,20 @@ import type { CasesClientArgs } from '..';
 import { LICENSING_CASE_ASSIGNMENT_FEATURE } from '../../common/constants';
 import { decodeOrThrow } from '../../../common/api/runtime_types';
 
+function validateCategory(category: string | null | undefined) {
+  if (category === undefined) {
```

Comment:
Do we need this? Validators seem to take care of it. Otherwise, you can change the type of the validators to be `category: string` and here do `if (category == null // take care both null and undefined) { return; }`